components/ReviewNotes.tsx

```diff
@@ -14,7 +14,15 @@ export const ReviewNotes: React.FunctionComponent<ReviewNotesProps> = (
   return (
     <>
       {notes.map((note, i) => (
-        <div key={i}>{note.value}</div>
+        <div key={i}>
+          {note.value}
+          {note.isPostVisitAction &&
+            (note.createdAt ? (
+              <div>Post visit action created on {note.createdAt}.</div>
```

Comment:
```suggestion
              <div>Post visit action created on {formatDate(note.createdAt, "d MMMM yyyy")}.</div>
```

Reply:
I've actually stored it as a formatted string - should I undo that?

Reply:
Oh yes. We should be storing it as an ISO timestamp.

Reply:
Nope, I'm being silly, we were doing that anyway - I just had to convert it into a date again to use `formatDate`